packages/cf-style-container/test/__snapshots__/index.js.snap

```diff
@@ -147,6 +147,8 @@ Object {
     },
     "color": Object {
       "apple": "#BD2527",
+      "apple10": "#f8e9e9",
```

Comment:
Sorry I missed this PR! 
I like this idea. What do you think of adding a comment about how the naming convention works. I'm assuming `apple10` is like saying `optacity(apple,0.1)`.

Reply:
Yeah that's exactly what it means. +1 to adding a note about the naming.